packages/design/package.json

```diff
@@ -3,56 +3,79 @@
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.mjs"
+    },
+    "./foundation.js": {
+      "types": "./dist/index.d.ts",
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.mjs"
+    },
+    "./foundation.css": {
+      "import": "./dist/foundation.css",
+      "require": "./dist/foundation.css"
+    },
+    "./dark.mode.css": {
+      "import": "./dist/dark.mode.css",
+      "require": "./dist/dark.mode.css"
+    },
+    "./colors": {
+      "import": "./dist/colors.mjs",
+      "require:": "./dist/colors.cjs"
+    },
+    "./colors.js": {
+      "import": "./dist/colors.mjs",
+      "require:": "./dist/colors.cjs"
+    }
+  },
   "files": [
     "dist/*",
     "*.css",
+    "*.mjs",
+    "*.cjs",
+    "*.js",
     "foundation.*",
     "icons/*"
   ],
   "scripts": {
-    "build": "npm run build:iconMap && npm run build:css && npm run build:css:icons && npm run build:colors && npm run build:cssTypes && npm run build:foundation && npm run build:iconCssStyle && npm run build:rollup",
-    "build:rollup": "rollup --config",
-    "build:css": "postcss src/foundation.css --output foundation.css --use postcss-import",
-    "build:css:icons": "postcss src/icons/*.css --dir icons/ --use postcss-import",
-    "build:ts": "tsc --project .",
-    "build:cssTypes": "tcm src/icons",
-    "build:colors": "node buildColors.js",
-    "build:foundation": "node buildFoundation.js && node resolveFoundationTokens.js && npm run build:foundationTypes && npm run build:addFoundationExceptions && npm run build:removeTempFiles",
-    "build:foundationTypes": "tsc --declaration --emitDeclarationOnly --allowJs foundation.js foundation.native.js foundation.android.js foundation.ios.js --outDir . --skipLibCheck",
-    "build:addFoundationExceptions": "./scripts/addFoundationExceptions.sh",
-    "build:removeTempFiles": "rm -f src/foundation.js && ./scripts/removeFoundationTempFiles.sh",
-    "build:iconMap": "ts-node --project ../../tsconfig.bin.json scripts/generateIconMap.ts",
-    "build:iconCssStyle": "node buildIconStyles.js && npm run build:iconCssStyleTypes",
-    "build:iconCssStyleTypes": "tsc --declaration --emitDeclarationOnly --allowJs src/icons/iconStyles.ts --outDir ./src/icons --skipLibCheck",
-    "clean": "rm -rf dist/* icons/* src/icons/iconMap.ts colors.js foundation.* iconStyles.* tsconfig.tsbuildinfo",
+    "test": "jest",
+    "build:design": "rollup -c rollup.config.mjs",
+    "generate:iconMap": "tsx src/scripts/buildIconMap.ts",
+    "generate:tokenCSS": "tsx src/scripts/buildTokens.ts",
+    "generate:assets": "npm run generate:iconMap && npm run generate:tokenCSS",
+    "build": "mkdir -p dist && npm run generate:assets && npm run build:design",
+    "watch": "mkdir -p dist & chokidar \"**/*.ts\" \"**/*.json\" \"**/*.svg\"-c \"npm run generate:assets && npm run build:design\"",
+    "clean": "rm -rf dist colors.js foundation.css src/assets colors.cjs colors.mjs node_modules",
```

Comment:
I am not sure if we want to delete `node_modules` since lerna's clean will do that and the clean script runs in prepare so node_modules are nuked in the initial `npm ci`